deploy/services/helm-charts/dss/values.example.yaml

```diff
@@ -1,5 +1,15 @@
 # See values.schema.json for schema description
 
+dss:
+  #  image: docker.io/interuss/dss:v0.15.0 # See https://hub.docker.com/r/interuss/dss/tags for official image releases.
```

Comment:
IMO in the example file it is OK to set the version (i.e. uncommenting this line).